packages/tools/webpack-fluid-loader/src/multiResolver.ts

```diff
@@ -51,6 +52,8 @@ function getUrlResolver(options: RouteOptions): IUrlResolver {
                     options.tenantId,
                     options.bearerSecret ?? "");
             }
+
+            options.fluidHost = options.fluidHost ?? "";
```

Comment:
Rather than mutating the input, it might be better to just create a local "fluidHost" variable and assign to that.